rpcs3/Emu/Audio/XAudio2/XAudio27Backend.cpp

```diff
@@ -7,189 +7,197 @@
 #include "XAudio2Backend.h"
 #include "3rdparty/XAudio2_7/XAudio2.h"
 
-static thread_local HMODULE s_tls_xaudio2_lib{};
-static thread_local IXAudio2* s_tls_xaudio2_instance{};
-static thread_local IXAudio2MasteringVoice* s_tls_master_voice{};
-static thread_local IXAudio2SourceVoice* s_tls_source_voice{};
-
-void XAudio2Backend::xa27_init(void* lib2_7)
+class XAudio27Library : public XAudio2Backend::XAudio2Library
 {
-	s_tls_xaudio2_lib = (HMODULE)lib2_7;
-
-	HRESULT hr = S_OK;
-
-	hr = CoInitializeEx(nullptr, COINIT_MULTITHREADED);
-	if (FAILED(hr))
-	{
-		LOG_ERROR(GENERAL, "XAudio2Backend : CoInitializeEx() failed(0x%08x)", (u32)hr);
-		Emu.Pause();
-		return;
-	}	
-
-	hr = XAudio2Create(&s_tls_xaudio2_instance, 0, XAUDIO2_DEFAULT_PROCESSOR);
-	if (FAILED(hr))
+	const HMODULE s_tls_xaudio2_lib;
+	IXAudio2* s_tls_xaudio2_instance{};
```

Comment:
s_tls without being static and thread local again

Reply:
Fixed.